packages/theme/src/components/donut-chart.ts

```diff
@@ -0,0 +1,36 @@
+import { mergeMultiStyle, type ComponentMultiStyle } from "@yamada-ui/core"
+import { LineChart } from "./line-chart"
+
+export const DonutChart: ComponentMultiStyle = mergeMultiStyle(LineChart, {
+  baseStyle: {
+    donut: {},
+    activeShape: {},
+    inactiveShape: {},
+    label: { fillOpacity: 1 },
+    labelLine: {},
+    cell: {},
+  },
+
+  sizes: {
+    sm: {
+      container: {
+        w: "full",
+      },
+    },
+    md: {
+      container: {
+        w: "full",
+      },
+    },
+    lg: {
+      container: {
+        w: "full",
+      },
+    },
+    full: {
```

Comment:
Since `lg` and `full` have the same layout, it would be better to delete `full`. At the same time, specify `full` in the `omit` array to exclude it.